package.json

```diff
@@ -8,15 +8,19 @@
     "type": "git",
     "url": "https://github.com/AtomLinter/linter-tslint.git"
   },
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "coffeelint ./lib"
+  },
   "license": "MIT",
   "engines": {
     "atom": ">0.50.0"
   },
   "dependencies": {
-    "atom-package-deps": "^4.0.1",
+    "atom-package-deps": "4.0.1",
```

Comment:
Greenkeeper will usually run the builds testing things within a few minutes, so as long as you are watching the notifications your area of vulnerability isn't that long.

If you pin the versions that's fine too (and as you note, slightly more stable), however just be warned that you will get PRs for every update.